Lib9c/GameConfig.cs

```diff
@@ -9,6 +9,7 @@ public static class GameConfig
 #endif
 
         public const int SlotCount = 3;
+        public const int ConsumableSlotCount = 5;
```

Comment:
Is this constant needed?

Reply:
yes.

Reply:
I guess it's only used in [test code] (https://github.com/planetarium/lib9c/pull/148/files#diff-bce870726bc5fa621b60324c6e112f4fb5f0d9ec21c9b9a3176b6e43cc77593bR155)... so it doesn't seem for Lib9c project right now.

Reply:
I added it to limit the number of consumable, but there is no using in `ValidateConsumable` right now. delete this.